http/http.go

```diff
@@ -0,0 +1,39 @@
+package http
```

Comment:
the package name `http` is bad which will cause a collision. Mb `api.Call()` insead of `http.Do()`?